### PR TITLE
Remove IE9 browser hacks.

### PR DESCRIPTION
**Pull request by bardiharborow**
Removes all the IE9 browser hacks I can find. Ref #21387.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -163,7 +163,6 @@ module.exports = function (grunt) {
 
     cssmin: {
       options: {
-        compatibility: 'ie10,-properties.zeroUnits',
```
Comment (bardiharborow) [inline Gruntfile.js]: `ie10` is not a valid clean-css option, and `-properties.zeroUnits` is no-longer needed now we're removing the relevant browser hack.

**Comment by Starsam80** [comment]
Could you also remove https://v4-alpha.getbootstrap.com/components/progress/#ie9-support?

**Comment by tomlutzenberger** [comment]
Pretty cool! 😀 

//cc @mdo: That's the stuff I told you about on Twitter.

**Comment by mdo** [comment]
@tomlutzenberger My bad :grin:

**Comment by mdo** [comment]
@Starsam80 I'll get that in a follow-up commit. Thanks!

**Pull request merged**